docs/historical/configuration.rst

```diff
@@ -47,3 +47,8 @@ the now (since Tahoe-LAFS v1.3.0) unsupported
 addresses specified in ``advertised_ip_addresses`` were used in
 addition to any that were automatically discovered), whereas the new
 ``tahoe.cfg`` directive is not (``tub.location`` is used verbatim).
+
+The stats gatherer has been broken at least since Tahoe-LAFS 1.13.0.
+The functionality of ``[client].stats_gatherer.furl`` (which was
```

Comment:
extra `.` at the beginning of `.stats_gatherer.furl` here.

Reply:
Removed that in 0e02f0c.